src/main/java/com/kodilla/ecommercee/domain/OrderItem.java

```diff
@@ -0,0 +1,45 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.jetbrains.annotations.NotNull;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Entity(name = "ORDER_ITEM")
+public class OrderItem {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @NotNull
+    @Column(name = "ORDER_ITEM_ID", unique = true)
+    private Long orderItemId;
+
+    @Column(name = "ORDER_STATUS")
+    private String orderStatus;
```

Comment:
As you mentioned in the chat, I think that 'Status' should only be in 'Order'. Moreover, it would be nice if it were a Boolean, as it would be easier to verify later.

Reply:
Let's wait for Andrzej's answer on chat.

Reply:
Andrzej confirmed that `Status` shouldn't be here, so I'll correct this.